lib/ts/recipe/totp/types.ts

```diff
@@ -0,0 +1,168 @@
+/* Copyright (c) 2023, VRAI Labs and/or its affiliates. All rights reserved.
+ *
+ * This software is licensed under the Apache License, Version 2.0 (the
+ * "License") as published by the Apache Software Foundation.
+ *
+ * You may not use this file except in compliance with the License. You may
+ * obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+import OverrideableBuilder from "supertokens-js-override";
+import { BaseRequest, BaseResponse } from "../../framework";
+import RecipeUserId from "../../recipeUserId";
+import { NormalisedAppinfo } from "../../types";
+import type { SessionContainer } from "../session";
+
+export type TypeInput = {
+    issuer?: string;
+    defaultSkew?: number;
+    defaultPeriod?: number;
+    getEmailOrPhoneForRecipeUserId?: GetEmailOrPhoneForRecipeUserIdFunc;
```

Comment:
since the arg in create device is called `userIdentifierInfo`, we should call this function getUserIdentifierInfoForUserId`. Also note that we do not call this function recipe user ID cause we do not want to pass it a recipe user ID, but instead the primary user ID

Reply:
change not made properly.

Reply:
Done now.